Imagine/Cache/CacheManager.php

```diff
@@ -138,10 +138,11 @@ public function getRuntimePath($path, array $runtimeConfig)
      * @param string $filter        The name of the imagine filter in effect
      * @param array  $runtimeConfig
      * @param string $resolver
+     * @param int    $referenceType The type of reference to be generated (one of the constants)
      *
      * @return string
      */
-    public function generateUrl($path, $filter, array $runtimeConfig = [], $resolver = null)
+    public function generateUrl($path, $filter, array $runtimeConfig = [], $resolver = null, $referenceType = UrlGeneratorInterface::ABSOLUTE_PATH)
```

Comment:
this is a BC break. the default value must be the same as was hardcoded before: ABSOLUTE_URL